api/src/main/java/org/pmiops/workbench/api/WorkspaceAdminController.java

```diff
@@ -72,4 +74,12 @@ public WorkspaceAdminController(WorkspaceAdminService workspaceAdminService) {
   public ResponseEntity<List<FileDetail>> listFiles(String workspaceNamespace) {
     return ResponseEntity.ok(workspaceAdminService.listFiles(workspaceNamespace));
   }
+
+  @Override
+  @AuthorityRequired({Authority.RESEARCHER_DATA_VIEW})
```

Comment:
The original method is:
```
  @Override
  @AuthorityRequired(Authority.SECURITY_ADMIN)
```

Not sure if this is correct here.

Reply:
It should be security admin. The other methods here do not mutate anything, they are read-only operations, hence the different authority.